cmd/statusd/main.go

```diff
@@ -33,6 +33,21 @@ var (
 	logLevel       = flag.String("log", "INFO", `Log level, one of: "ERROR", "WARN", "INFO", "DEBUG", and "TRACE"`)
 	logFile        = flag.String("logfile", "", "Path to the log file")
 	version        = flag.Bool("version", false, "Print version")
+	les            = flag.Bool("les", false, "LES protocol")
+
+	listenAddr   = flag.String("listenaddr", ":30303", "IP address and port of this node (e.g. 127.0.0.1:30303)")
+	identity     = flag.String("shh.identity", "", "Protocol identity file (private key used for asymmetric encryption)")
+	passwordFile = flag.String("shh.passwordfile", "", "Password file (password is used for symmetric encryption)")
+	standalone   = flag.Bool("standalone", true, "Don't actively connect to peers, wait for incoming connections")
```

Comment:
Maybe we could group `shh` related flags into one group? `standalone` is kind of put here in between.

Reply:
i was planning on doing that but was waiting on confirmation that both flag listenaddr and flag standalone where not exclusively applicable to whisper.

Reply:
No, both `listenaddr` and `standalone` are independent.

Reply:
okay I'll move standalone above shh stuff ... that way all shh are grouped together

what about codeclimate barking ?  I asked about ?